GUI/CalibrateTaskWindow.xaml.cs

```diff
@@ -270,5 +272,50 @@ private void KeyPressed(object sender, KeyEventArgs e)
                 CancelButton_Click(sender, e);
             }
         }
+
+        internal SearchModifications SearchMod = new SearchModifications();
+
+        private void TextChanged_Fixed(object sender, TextChangedEventArgs args)
+        {
+            SearchMod.FixedSearch = true;
+            SetTimer();
+        }
+
+        private void TextChanged_Var(object sender, TextChangedEventArgs args)
+        {
+            SearchMod.VarSearch = true;
+            SetTimer();
+        }
+
+        private void SetTimer()
+        {
+            if (!SearchMod.TimerCreated)
+            {
+                SearchMod.Timer.Tick += new EventHandler(TextChangeTimerHandler);
+                SearchMod.TimerCreated = true;
+            }
+            SearchMod.SetTimer();
+        }
+
+        private void TextChangeTimerHandler(object sender, EventArgs e)
```

Comment:
I don't think you need this method if you call `SearchModifications.FilterTree` above